.github/workflows/ci_tests.yaml

```
@@ -107,8 +112,8 @@ jobs:
             gmt=6.5.0
             ghostscript=10.02.1
             numpy=${{ matrix.numpy-version }}
-            pandas
-            xarray
+            pandas=${{ matrix.pandas-version }}
+            xarray=${{ matrix.xarray-version }}
```

Remove equal-to sign here
```suggestion
            pandas${{ matrix.pandas-version }}
            xarray${{ matrix.xarray-version }}
```